orienteer-users/src/main/java/org/orienteer/users/module/OrienteerUsersModule.java

```diff
@@ -87,8 +87,8 @@ private void updateOrienteerUserRoleDoc(ODatabaseDocument db, ODocument perspect
 
         role.grant(OSecurityHelper.FEATURE_RESOURCE, SearchPage.SEARCH_FEATURE, OrientPermission.READ.getPermissionFlag());
 
-        role.grant(ResourceGeneric.CLASS, OrienteerUser.CLASS_NAME, OrientPermission.UPDATE.getPermissionFlag());
-        role.grant(ResourceGeneric.DATABASE, "cluster", OrientPermission.UPDATE.getPermissionFlag());
+        role.grant(ResourceGeneric.CLASS, OrienteerUser.CLASS_NAME, 6);
```

Comment:
It's hard to get what 6 actually means. Recommend to use OrientPermission - there is  method to get combined flag as int.